source/dem/find_contact_detection_frequency.cc

```diff
@@ -0,0 +1,69 @@
+#include <dem/find_contact_detection_frequency.h>
+
+using namespace dealii;
+
+template <int dim>
+bool
+find_contact_detection_frequency(
```

Comment:
I don't think this is the right name. This function finds if contact detection is necessary, not the frequency.